environment.yml

```diff
@@ -52,3 +52,5 @@ dependencies:
   - pytest >=4.4
   - pytest-cov >=2.6
   - requests-mock >=1.8.0
+  - rasterio
+  - rioxarray
```

Comment:
Please add minimum version. First guess is to use the currently installed version's major and mnor version number e.g. ` >=x.y`